### ISSUE TITLE
Write up game rules

### ISSUE THREAD

**Issue by kingdomlevel** [kingdomlevel/toast#3]
We should finish writing up the game rules in the [project overview doc](https://docs.google.com/document/d/1XYE5PjRK1ZJRHjGWacaso1-OAbhQGMu-BHBEX5lOUpg/edit#).

I made a wee start on this on the first day but we should try and formalise the full game rules before starting on dev _proper_.

**Comment by johnpooley** [comment]
I think these rules look clear and comprehensive enough to get the game going. We may need to refine them once we are able to playtest it, but this should be enough to start building the app.

Maybe we can add an issue to decide scoring to the backlog and mark this as done.

**Comment by johnpooley** [comment]
I've copied this comment from Slack, not sure if this should be its own issue or should be added to rules or wireframes?

> heres an idea, if we are going for a fixed number of rounds/turns for the end of the game, maybe the loaf of bread could represent all the remaining songs?
> 
> so when it's your turn you get a slice of bread with 3 songs written on it and you toast it for the length of the timer, it pops out of the toaster with the correct answer on it for all to see, rather than everyone has a loaf

**Comment by kingdomlevel** [comment]
> I've copied this comment from Slack, not sure if this should be its own issue or should be added to rules or wireframes?
> 
> > heres an idea, if we are going for a fixed number of rounds/turns for the end of the game, maybe the loaf of bread could represent all the remaining songs?
> > so when it's your turn you get a slice of bread with 3 songs written on it and you toast it for the length of the timer, it pops out of the toaster with the correct answer on it for all to see, rather than everyone has a loaf

i will reference this in the wireframes issue - for now marking this as closed.
